providers/base/units/camera/jobs.pxu

```diff
@@ -102,11 +102,29 @@ _summary: Webcam multiple resolution capture test for {product_slug}
 estimated_duration: 1.2
 depends: camera/detect
 command:
-  camera_test.py resolutions -d /dev/{name}
+  camera_test.py resolutions -d /dev/{name} -o "$PLAINBOX_SESSION_SHARE"
 _description:
   Takes multiple pictures based on the resolutions supported by the camera and
   validates their size and that they are of a valid format.
 
+unit: template
+template-resource: device
+template-filter: device.category == 'CAPTURE' and device.name != ''
+template-unit: job
+plugin: attachment
+category_id: com.canonical.plainbox::camera
+id: camera/multiple-resolution-images-attachment_{name}
+flags: also-after-suspend
+_summary: Attach an image from the multiple resolution images test for {product_slug}
+estimated_duration: 1s
+after: camera/multiple-resolution-images_{name}
+command:
+  [ -f "$PLAINBOX_SESSION_SHARE"/resolution_test_image_{name}.jpg ] &&
```

Comment:
If you adopt the change (n at the end) to the `resolution_test_image_{name}_*.jpg`, consider updating this as well